web/src/locales/zh/data.json

```diff
@@ -14,14 +14,20 @@
     "The system has detected that you are using the default password, which is not secure. You need to modify your password immediately. Here are the instructions": "系统检测到您正在使用默认密码，存在安全隐患。请立即修改您的密码。以下是具体操作"
   },
   "application": {
+    "Deploy": "部署",
+    "DisConnect": "断开连接",
```

Comment:
Use `general:Active` and `general:Inactive`, add these, together with other code:

<img width="1702" height="478" alt="image" src="https://github.com/user-attachments/assets/6f140827-4117-447e-9211-09e4a3760856" />